lib/core/src/Cardano/Wallet/Api/Client.hs

```diff
@@ -330,9 +330,10 @@ addressClient =
                 fmap unApiAddressInspect
                 . _inspectAddress
                 . ApiAddressInspectData
-            , postRandomAddress = \_ _ -> fail "feature unavailable."
-            , putRandomAddress  = \_ _ -> fail "feature unavailable."
-            , putRandomAddresses = \_ _ -> fail "feature unavailable."
+            , postRandomAddress = \_ _ -> error "feature unavailable."
+                -- TODO: Fix. Do we want a JSON error response?
```

Comment:
I suspect we don't want `error` here

Reply:
Neither `error` nor `fail`.

`UnliftIO.Exception.throwString` is more explicit, yet still convenient to use in these circumstance.